### PR TITLE
Add drop_partition,release_partitions,load_partitions

### PR DESCRIPTION
**Pull request by kingzcheung**
See #55

### CODE REVIEW

File: src/collection.rs [hunk 1]
```diff
@@ -323,9 +329,64 @@ impl Collection {
...
+                    collection_name: self.schema().name.to_string(),
+                    partition_name: name.as_ref().to_owned(),
+                })
+                .await?
+                .into_inner(),
+        ))
+    }
+
+    /// Load Partitions
+    pub async fn load_partitions<S: ToString, I: IntoIterator<Item = S>>(
```
Comment (yah01) [inline src/collection.rs:349]: Ref line 131, the method need a `replica_number` parameter

File: src/collection.rs [hunk 2]
```diff
@@ -323,9 +329,64 @@ impl Collection {
...
+                .load_partitions(LoadPartitionsRequest {
+                    base: Some(new_msg(MsgType::LoadPartitions)),
+                    db_name: "".to_string(),
+                    collection_name: self.schema().name.to_string(),
+                    partition_names: names.into_iter().map(|x| x.to_string()).collect(),
+                    replica_number: 1,
+                })
+                .await?
+                .into_inner(),
+        ))
```
Comment (yah01) [inline src/collection.rs:365]: we have to wait the load done, `Load_Xxx` method of Milvus only commits the request, use `get_loading_progress` to make sure the collection/partitions available

**Review by yah01 — CHANGES_REQUESTED** [review]

File: src/collection.rs (hunk 1 above)
Comment (kingzcheung) [reply src/collection.rs:349]: fixed it.

**Comment by kingzcheung** [comment]
well, my bad, I re-impl ...

**Review by yah01 — APPROVED** [review]
/LGTM

**Pull request merged**